luarules/gadgets/game_end.lua

```diff
@@ -340,9 +334,18 @@ if gadgetHandler:IsSyncedCode() then
 
 	function gadget:GameFrame(gf)
 		if gameoverFrame then
+			if not globalLosGranted then
```

Comment:
why not use `gadget:GameOver`?

Reply:
`gameoverFrame` is set once the win condition is fulfilled, but is in the future. `GameOver` takes place after everything chain explodes, so is too late.